files/initramfs-tools/arista-convertfs.j2

```diff
@@ -224,7 +224,7 @@ cmd="wait_for_root_dev"
 run_cmd "$cmd" "$err_msg"
 
 err_msg="Error: formatting to ext4 failed"
-cmd="mke2fs -t ext4 -F -O '^huge_file,^metadata_csum' $root_dev"
+cmd="/usr/local/sbin/mke2fs -t ext4 -F -O '^huge_file,^metadata_csum' $root_dev"
```

Comment:
>/usr/local/sbin/ [](start = 5, length = 16)

Is it related to buster? If not, separate it into another PR?

Reply:
check explanation in the commit message. https://github.com/Azure/sonic-buildimage/pull/4405/commits/8092e9826b167cc84378564659e5e5770b04d87a

/usr/sbin/mke2fs is now from busybox which does not support
the operation we need